docs/gitlab.md

```diff
@@ -150,19 +150,13 @@ The output consists of the following:
 - An overview of all ratings, compared against the system as a whole. This allows you to check if your changes improved the system, or accidentally made things worse.
 - The final conclusion on whether your changes and merge request meet the quality target.
 
-In addition to the textual output, Sigrid CI also generates a static HTML file that shows the results in a more graphical form. This is similar to test coverage tools, which also tend to produce a HTML report. You can download this report form GitLab using the "download" button on the right side of the CI results view:
+In addition to the textual output, the results also include a link to the Sigrid landing page. 
 
-<img src="images/gitlab-download.png" width="300" />
+<img src="images/landing-page.png" width="700" />
 
-The information in the HTML report is based on the aforementioned list, though it includes slightly more detail.
+You can open this URL in order to use Sigrid for interpreting your analysis results. Whether you should use the text output or the Sigrid page is largely down to personal preference: the text output is faster to acces and more concise, while Sigrid allows you to view results in a more visual and interactive way. 
```

Comment:
The link to the Sigrid landing page is technically part of the textual output (contrary to what was mentioned in the old text). I wanted to suggest adding it to the list as another bullet but I think it's clearer to indeed have this text separated from the list. I would however change the text accordingly. I would also move the first sentence of the next paragraph here. The result would be the following:

```
At the end of the textual output there is a link to the Sigrid landing page. You can open this URL in order to use Sigrid for interpreting your analysis results.

<img src="images/landing-page.png" width="700" />

Whether you should use the text output or the Sigrid page is largely down to personal preference: the text output is faster to acces and more concise, while Sigrid allows you to view results in a more visual and interactive way. 
```

(I can't seem to put this as a suggestion directly as GitHub won't allow those on removed lines and even in split mode I can't only select the new and unchanged lines...)